### DISCUSSION TITLE
Eslint v9 support for `@remix-run/eslint-config`

### DISCUSSION THREAD

**Discussion by RobertGemmaJr** [remix-run/remix#10054] [Proposals]
Remix is one of the biggest tools in my project(s) that I'm stilling waiting on for an upgrade to Eslint v9. Any ideas on a timeline for the implementation?

**Comment by sergiodxa** [comment]
The ESLint config was deprecated, new projects doesn't come with it anymore and comes with the config directly in the `. eslintrc.cjs` file, so I doubt they will upgrade it to ESLint v9.

**Reply by HenrikFricke** [reply]
I ran `npx create-remix@latest` today and still got ESLint version 8.57.1. Do you have any plans to update the template and provide an ESLint config with v9 support? Or will you remove ESLint from the template?

**Reply by andrew-secret** [reply]
Any updates on this one? @sergiodxa

**Reply by sergiodxa** [reply]
The templates comes without the `@remix-run/eslint-config` and has come without that for a long time, you can check templates here https://github.com/remix-run/remix/tree/main/templates

The `@remix-run/eslint-config` has been deprecated for a while now.

I'm not sure when or if the team will update the default config to v9, but since you own it you can do the upgrade after you create a new app, you're not blocked by the `@remix-run/eslint-config`.
